src/Features/Core/Portable/AddParameter/AbstractAddParameterCodeFixProvider.cs

```diff
@@ -392,7 +392,7 @@ private static async Task<(ITypeSymbol, RefKind)> GetArgumentTypeAndRefKindAsync
             var semanticModel = await invocationDocument.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
             var argumentExpression = syntaxFacts.GetExpressionOfArgument(argument);
             Contract.ThrowIfNull(argumentExpression);
-            var argumentType = semanticModel.GetTypeInfo(argumentExpression, cancellationToken).Type ?? semanticModel.Compilation.ObjectType;
+            var argumentType = semanticModel.GetType(argumentExpression, cancellationToken) ?? semanticModel.Compilation.ObjectType;
```

Comment:
feels like we should be calling through to InternalExtensions.DetermineParameterType.  That's what generate-method uses, and this would keep them consistent :)

Reply:
`InternalExtensions` is C# specific (`MS.CA.CSharp`). Using it will require to extract this functionality to an abstract method, and there is no such helper analog for VB, so we need either to use non-specifit `GetType`, which eliminates all sense in doing all the ceremony, or create such helper, which is just a lot of code, and all of that is just for consistency reasons. Do we _really_ need that to be done?

Reply:
If we're going to make this change here, it seems like we shoudl do the work to make the features consistent.  

> Using it will require to extract this functionality to an abstract method

That seems fine :)

> and there is no such helper analog for VB

Isn't there ArgumentSyntaxExtensions.DetermineType?

Reply:
> Do we really need that to be done?

@DoctorKrolic If you'd prefer, i'd be happy to make this change.  LMK! :)

Reply:
Done myself. Please review

> Isn't there ArgumentSyntaxExtensions.DetermineType?

Yes, there is. Very helpful info

Reply:
yw :)